docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,48 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode network that can be used as a simple replacement for Slirp.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          passt: {} # connect using passt mode
+  networks:
+  - name: red
+    pod: {}
+```
+
+#### Requirements:
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
```

Comment:
Could you note how much RAM one VM with all ports open takes? And how much for a VM requesting a single port. To motivate users to be specific

Reply:
Regarding the exact RAM gap. I don't know,
But it is mentioned in the doc that ``vmi.Spec.Domain.Resources.Requests.Memory` should be set to at least `2048` when running passt VM with no explicit ports.
For "regular" VMs we usually request 1024.

@phoracek it you think more precise values are needed, it would require further investigation.
Can be done as a follow-up.

Reply:
> Regarding the exact RAM gap. I don't know, But it is mentioned in the doc that ``vmi.Spec.Domain.Resources.Requests.Memory`should be set to at least`2048` when running passt VM with no explicit ports. For "regular" VMs we usually request 1024.
> 
> @phoracek it you think more precise values are needed, it would require further investigation. Can be done as a follow-up.

I agree on the follow-up ... This is highly experimental, and users can always open up issues requesting us to clarify some parts.